services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/exceptions/HttpErrors.java

```diff
@@ -0,0 +1,59 @@
+package com.sourcefuse.jarc.services.usertenantservice.exceptions;
+
+public class HttpErrors {
+    public static HttpError HttpError(String message) {
```

Comment:
Check auth service PR for handling HTTP Errors

Reply:
done